guides/common/modules/proc_creating-a-suse-operating-system.adoc

```diff
@@ -0,0 +1,33 @@
+[id="Creating_a_SUSE_Operating_System_{context}"]
+= Creating a SUSE Operating System
+
+{ProvisioningDocURL}adding-installation-media_provisioning[Create an installation media] and {ProvisioningDocURL}creating-operating-systems_provisioning[operating system] entry for SLES 15 SP3.
+
+.Prerequisite
+* An xref:Preparing_SUSE_Installation_Media_{context}[extracted `.iso` image] on {Project} or an HTTP server that can be reached during the host provisioning process.
+
+.Procedure
+. Navigate to *Hosts > Installation Media* and click *Create Medium*.
+** The *Name* identifies the installation media for SLES 15 SP3.
```

Comment:
Changed all `** ` to `. `. In this specific example, it's imporant for the user to add `local` to the installation medium name because it is actually "local", i.e. on their Project Server.

Changed to more active wording; e.g. from ~The Name identifies~ to `set a Name`.